src/store/index.js

```diff
@@ -49,5 +50,10 @@ export default new Vuex.Store({
       })
     }
   },
+  getters: {
```

Comment:
If you use a computed property to access the state, this getter is not really necessary.

Reply:
Thanks, @DietBepis1